noxfile.py

```diff
@@ -123,7 +132,7 @@ def cover(session):
         "--cov=google.oauth2",
         "--cov=tests",
         "--cov=tests_async",
-        "--cov-report=",
+        "--cov-report=term-missing",
```

Comment:
Why?

Reply:
If lines aren't being covered, then the report should include them, so that the hapless developer can figure out why.  I've never understood why that isn't the default behavior for the report.